pkg/values/bool.go

```diff
@@ -0,0 +1,21 @@
+package values
+
+import (
+	"github.com/smartcontractkit/chainlink-common/pkg/values/pb"
+)
+
+type Bool struct {
+	Value bool
```

Comment:
this had me confused for minute. here `Value` is a field name. other places `Value` is type.

1. does this field need to be public? it seem like the intention is to use the interface funcs, not the embedded field
2. can you change the name of the field to something else, maybe impl or data?

the readibility is even more difficult for me in the Map type ~ Value map[any]Value

Reply:
Yes, can rename :) I think leaving it public is valuable since if you type coerce it you can then get access to the underlying typed value.